portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/**
+ * Class representing a comment that users add to the website.
+ */
+public class Comment {
```

Comment:
Prefer to make classes `final` unless you intend to create inheritance. 

In most cases, you'd create inheritance by having an `interface` and then having one or more `final class`es implement it.

Reply:
done